src/server/json_family.cc

```diff
@@ -21,9 +21,12 @@
 #include "core/json/path.h"
 #include "facade/cmd_arg_parser.h"
 #include "facade/op_status.h"
+#include "facade/reply_builder.h"
+#include "glog/logging.h"
```

Comment:
no need for glog/logging, please disable "include what you see" in your clang checker